Server/mods/deathmatch/logic/CGame.cpp

```diff
@@ -1410,6 +1410,12 @@ void CGame::QuitPlayer(CPlayer& Player, CClient::eQuitReasons Reason, bool bSayI
     // If he had joined
     if (Player.IsJoined())
     {
+        // Check if the player was inside a team so we could fire onPlayerTeamChange event
+        if (Player.GetTeam())
```

Comment:
As suggested, use a variable instead:
```cpp
if (CTeam* pPlayerTeam = Player.GetTeam())
```

Reply:
I called ``Player.GetTeam()`` twice on my previous pull request, which I changed now. So, I don't think I should declare a variable for a one time use no?

P.S: Sorry for taking too long, will work on this as soon as possible.

```
// Check if the player was inside a team so we could fire onPlayerTeamChange event
if (Player.GetTeam())
{
    Player.SetTeam(NULL, false, CTeam::CHANGE_DISCONNECTED);
}
```